### PR TITLE
added docs for "Vue.use" options

### PR DESCRIPTION
**Pull request by zoul0813**
introduced in https://github.com/vue-generators/vue-form-generator/pull/560

### CODE REVIEW

**Review by icebob — APPROVED** [review]
@zoul0813 I think you should not wait for us to approve all changes. Currently you are the lead maintainer :)

**Comment by zoul0813** [comment]
@icebob just looking for some feedback, and giving everyone the opportunity to chime in ... I figured I'd wait 2 days for PR's and then just merge them in.  On holiday for the next 3 weeks, so hopefully I'll give some VFG issues cleared out and finish migrating/updating the docs to the new version of GitBooks.

**Pull request merged**
